framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -123,6 +128,14 @@ qx.Class.define("qx.bom.webfonts.Validator", {
       apply : "_applyFontFamily"
     },
 
+    /**
+     * Comparison string used to check whether the font has loaded or not.
+     */
+    comparisonString :
+    {
+      nullable : true,
+      init : null
```

Comment:
There was no special reason for that. Static storage is once, property for every instance. There's only one place in the code where it is or'ed to the default.